routes/web.php

```diff
@@ -13,6 +14,6 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', [AdminController::class, 'index'])->name('index');
+
+Route::get('story/{id}', [AdminController::class, 'story'])->name('story');
```

Comment:
- Ook hier zou ik de pagina achter admin zetten.

- Daarnaast is {id} niet de meest beschrijvende variabel naam in dit geval. Ik denk dat {story} hier meer gepast is. Maar volgens mij word het vaker gebruikt ook bij andere, dus ben ik misschien wel de zeikerd.

- dan zou de route zo worden : 'admin/stories/{story}'